src/com/ableton/VirtualEnv.groovy

```diff
@@ -5,10 +5,14 @@ class VirtualEnv implements Serializable {
   def script
   def destDir
 
-  VirtualEnv(script) {
+  VirtualEnv(script, python) {
     this.script = script
     def pathSeparator = script.isUnix() ? "/" : "\\"
-    this.destDir = script.pwd(tmp:true) + pathSeparator + script.env.BUILD_NUMBER
+    this.destDir = script.pwd(tmp:true) +
```

Comment:
FYI @ala-ableton, we should actually be putting `+` on the end of lines, not at the beginning of lines. Because Groovy doesn't have a line ending character (like `;`), having `+` at the beginning of the next line can cause some weird syntax errors.